kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -3014,6 +3006,15 @@ def cleanup_addons():
                     path = chan.source.path_
                     linux.rm_file_force(path)
 
+            for shmem in self.domain_xmlobject.devices.get_child_node_as_list('shmem'):
+                name = getattr(shmem, 'name_', None)
+                if not name:
+                    name = getattr(shmem, 'name', None)
+                if not name:
+                    continue
+
+                linux.rm_file_force(os.path.join('/dev/shm', name))
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)